.github/workflows/notifier-cd.yml

```
@@ -0,0 +1,29 @@
+# Notifierのデプロイ
+name: notifier-CD
+
+on: workflow_dispatch
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
```

こんな感じでマージ後に自動デプロイでヨシ！！
```suggestion
on:
  pull_request:
    branches:
      - main
    types: [closed]

jobs:
  deploy:
    runs-on: ubuntu-latest
    if: github.event.pull_request.merged == true
```

とりあえず動くかみる場合はPRだけで実行できるように書き換えて、動くまで修正するのもあり(普段はそうやって検証してる)
```
  pull_request:
    paths:
      - "notifier/**"
```

---

ありがとうございます！！
検証してからPRにします！

---

動作確認できました！
https://github.com/coniva-bouldering/bouldering-gym-notifier/actions/runs/5272787221